### PR TITLE
feat: way to skip the generation of last modification time

### PR DESCRIPTION
**Pull request by bfabio**
When using `chore(content-skip)` somewhere in the commit message for commits that change src/data/content/, the last modification time in the generated pages won't be updated.

### CODE REVIEW

**Comment by bfabio** [comment]
Depends on https://github.com/CollierCZ/gatsby-transformer-gitinfo/pull/212

**Comment by bfabio** [comment]
@astagi @Fupete 
1. What language do we want the README to be in?
2. We can change `chore(content-skip)` to something else, if we want. The upside is that it can be used as a conventional commit, but the downsides are that it's maybe difficoult for editors and `content-skip` is not super descriptive (or is it?)

**Comment by bfabio** [comment]
(we'll remove the fake content commit and the one depending on my repo before merging)

**Comment by bfabio** [comment]
Upstream [merged my patch](https://github.com/CollierCZ/gatsby-transformer-gitinfo/pull/212), so we can use it instead of my forked repo.

File: src/data/content/index.yaml [hunk 1]
```diff
@@ -58,7 +58,7 @@ components:
     big: true
     specular: true
     reversedMobile: false
-    title: Partecipa alla trasformazione digitale del Paese #C
+    title: Foobar #C
```
Comment (Fupete) [inline src/data/content/index.yaml:61]: ⚠️❓ @bfabio

Comment (bfabio) [reply src/data/content/index.yaml:61]: See https://github.com/italia/designers.italia.it/pull/898#issuecomment-1735527293

**Comment by bfabio** [comment]
Changed to `(last-update-skip)`, so it can be used with any conventional commit type (idea by @Fupete), in addition to the message body.

**Review by Fupete — APPROVED** [review]

**Pull request merged**